README.md

```diff
@@ -111,6 +112,24 @@ output.
 | `Images` | no | Used to map a given update image to an entry in `config.yaml`. There may be multiple entries that have the same `SourceImage`, but different `TargetImageName`s, so we need to choose which one receives the update image.
 | `ImageDenylist` | no | A list of images to exclude from the results.
 
+#### `Registry`
+
+The `Registry` fetches all image tags that matches the VersionFilter from a registry defined in the Images provided.
+Supported registries are: 
+* Suse Container Registry (registry.suse.com)
+* Docker Hub
+* Quay.io
+* K8s registry (registry.k8s.io)
+* GitHub Container Registry (ghcr.io)
+* Google Container Registry (gcr.io)
+
+| Field           | Required | Description |
+|-----------------|----------|------------- |
+| `Images`        | yes      | Used to map a given update image to an entry in `config.yaml`. There may be multiple entries that have the same `SourceImage`, but different `TargetImageName`s, so we need to choose which one receives the update image.
+| `Latest`        | no       | A flag to only use the latest tag.
```

Comment:
We should probably say something about it finding the latest tag via semver here. Not every tag is in semver format.